console.error("HALP!");
console.log("The second error has been resolved!");
console.warn("The third error has been resolved!")
